Src/StdLib/Lib/types.py

```diff
@@ -38,7 +38,7 @@ def _m(self): pass
 
 # For Jython, the following two types are identical
 GetSetDescriptorType = type(FunctionType.__code__)
-MemberDescriptorType = type(FunctionType.__globals__)
+MemberDescriptorType = type(ModuleType.__dict__["__dict__"]) # ironpython: type(FunctionType.__globals__) is getset_descriptor
```

Comment:
From the comment above it seems that Jython has the same problem as IronPython. Maybe submit a patch upstream?

Reply:
> Maybe submit a patch upstream?

Definitely worth considering. I'm actually not sure what the technical differences are between the two types. Most of out descriptors seem to be of the type `getset_descriptor`. The codebase does have `member_descriptor` so I just poked around until I found one, but I don't know what makes this one special...